modules/phase_field/include/postprocessors/GrainTracker.h

```diff
@@ -44,14 +46,17 @@ class GrainTracker : public NodalFloodCount
    * Returns a list of active unique grains for a particular node in a vector of pairs
    * (unique_grain_id, variable_idx)
    */
-  virtual const std::vector<std::pair<unsigned int, unsigned int> > & getNodalValues(dof_id_type node_id) const;
+//  virtual const std::vector<std::pair<unsigned int, unsigned int> > & getNodalValues(dof_id_type node_id) const;
```

Comment:
Dang, that's a loaded question...
Yes... then why don't you remove it?
No... then fix it

It probably should be removed but I might restore it if Mike wants it back...  Figured this was ok